main/management/commands/start.py

```diff
@@ -31,9 +32,22 @@ def handle(self, *args, **options):
             logging.error('Database could not be found, exiting.')
             sys.exit(1)
 
-        print('Running migrations')
-        call_command("migrate")
-        print('Running setup')
+        attempts_left = 5
+        while attempts_left:
+            try:
+                logging.info('Trying to run migrations')
+                call_command("migrate")
+                logging.info('Migrations ran')
+                break
+            except OperationalError:
+                attempts_left -= 1
+                logging.warning('Cannot run migrations, retrying.')
+                time.sleep(0.5)
+        else:
+            logging.error('Migrations could not be run, exiting.')
+            sys.exit(1)
```

Comment:
`sys.exit` will actually happily accept a human readable reason, so here you might want to do `sys.exit("Migrations unsuccessful")` or something.